core/src/main/scala/tech/beshu/ror/accesscontrol/blocks/variables/runtime/RuntimeResolvableVariable.scala

```diff
@@ -136,6 +137,19 @@ object SingleExtractable {
       }
     }
   }
+
+  case object CurrentGroupVar extends SingleExtractable with VariableType.CurrentGroup {
+
+    override def extractUsing(requestContext: RequestContext,
+                              blockContext: BlockContext): Either[ExtractError, String] = {
+      blockContext
+        .currentGroup
+        .map(_.value.value) match {
+        case Some(value) => Right(value)
+        case None => Left(ExtractError(s"Cannot extract user current group from block context"))
```

Comment:
IMO the message should be changed to sth like this:
"There was no current group found for request: ${requestContext.id}"